RespiratoryAssistDevices/R4/resources/Questionnaire-R4-RespiratoryAssistDevicesLab.json

```diff
@@ -0,0 +1,789 @@
+{
+  "resourceType": "Questionnaire",
+  "id": "RespiratoryAssistDevicesLab",
+  "name": "RespiratoryAssistDevicesLab",
+  "meta": {
+    "profile": [
+      "http://hl7.org/fhir/StructureDefinition/cqf-questionnaire",
+      "http://hl7.org/fhir/us/davinci-dtr/StructureDefinition/dtr-questionnaire-r4"
+    ]
+  },
+  "title": "Respiratory Assist Device Laboratory Test Results",
+  "status": "draft",
+  "subjectType": [
+    "Patient"
+  ],
+  "date": "2020-07-08",
+  "publisher": "Da Vinci DTR",
+  "extension": [
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/RespiratoryAssistDevices-prepopulation"
+    },
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicClinicalInfo-prepopulation"
+    }
+  ],
+  "item": [
+    {
+      "linkId": "1",
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/sub-questionnaire",
+          "valueCanonical": "questionnaire/patient-info"
+        }
+      ],
+      "type": "display"
+    },
+    {
+      "linkId": "2",
+      "type": "group",
+      "text": "Provider (physician/NPP) who is performing the face-to-face evaluation:",
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+          "valueCodeableConcept": {
+            "coding": [
+              {
+                "system": "http://hl7.org/fhir/questionnaire-item-control",
+                "code": "gtable"
+              }
+            ]
+          }
+        }
+      ],
+      "item": [
+        {
```

Comment:
put subqestionnaire for item 2.1 to 2.4

Reply:
This is the Lab form. I am pulling the encounter date from the DeviceRequest.encounter.reference to get the encounter date. We can't pull the encounter provider's names etc., so left it blank to be filled.